nbs/core.ipynb

```diff
@@ -59,13 +59,25 @@
     "import warnings\n",
```

Comment:
<p>Can you add some intermediate comments here please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/829/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>